decidim-consultations/app/models/decidim/consultation.rb

```diff
@@ -76,5 +76,9 @@ def self.order_randomly(seed)
         select('"decidim_consultations".*, RANDOM()').order(Arel.sql("RANDOM()")).load
       end
     end
+
+    def can_participate?(_user)
```

Comment:
Should we move this to the `Decidim::Parrticipable` concern as the default method? This way other participatory spaces can redefine it it they need to, but we ensure every par5ticipatory space responds to this method.